common/network-common/src/main/java/org/apache/spark/network/server/OneForOneStreamManager.java

```diff
@@ -95,6 +97,25 @@ public ManagedBuffer getChunk(long streamId, int chunkIndex) {
   }
 
   @Override
+  public ManagedBuffer openStream(String streamChunkId) {
```

Comment:
@jinxing64 this breaks old shuffle service. We should avoid change server side codes. Right now I just disabled this feature in #18467